packages/sui-helpers/cli.js

```diff
@@ -1,5 +1,5 @@
 /* eslint no-console:0 */
-const processSpawn = require('child_process').spawn
+const processSpawn = require('cross-spawn')
```

Comment:
the majority of our commands that call commands in sui use sui-helpers. So this should solve a lot of windows related errors.